.github/workflows/import_paths.yml

```
@@ -0,0 +1,77 @@
+# This is a manua workflow that does the following when trigerred:
+# - Runs a script to find and replace Go import path major version with given version.
+# - Commits and pushes changes to the source-branch.
+# - Opens a PR from the source branch to the target-branch.
+
+name: Update Go Import Paths
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Current Version that we want to change'
+        default: '11'
```

```suggestion
        default: '10'
```

Wdyt about keeping main at 10